Makefile

```
@@ -14,8 +14,8 @@ ifeq ($(project), playground)
 	cd packages/playground && yarn dev
 else ifeq ($(project), dashboard)
 	cd packages/dashboard && yarn serve
-else ifeq ($(project), UI)
-	cd packages/UI && yarn dev
+else ifeq ($(project), stellar_solana_bridge)
+	cd packages/stellar_solana_bridge && yarn serve
```

I see it's `dev` in package.json
```suggestion
	cd packages/stellar_solana_bridge && yarn dev
```